src/pages/Login.jsx

```diff
@@ -0,0 +1,16 @@
+import React, { useState } from "react";
+import LoginForm from "../components/LoginForm/LoginForm";
+
+const Login = () => {
+  const [errorMessage, setErrorMessage] = useState("");
+  return (
+    <>
+      <LoginForm
```

Comment:
IMO, we should implement the LoginForm here as this component is not doing anything rather than importing the form, so it makes things redundant creating extra files in the project

Reply:
For designing the login form, I take it as a component as there are no .scss files in my page folder.